checks.d/mongo.py

```diff
@@ -141,6 +141,9 @@ class MongoDb(AgentCheck):
         "opcountersRepl.insert": RATE,
         "opcountersRepl.query": RATE,
         "opcountersRepl.update": RATE,
+        "oplog.totalSizeMB": GAUGE,
```

Comment:
Can we name it `oplog.logSizeMB` instead, to match  https://docs.mongodb.org/v3.2/reference/method/db.getReplicationInfo/ ?